packages/tpetra/core/src/Tpetra_Details_local_deep_copy.hpp

```diff
@@ -0,0 +1,117 @@
+// @HEADER
```

Comment:
How does this reimplementation differ from Kokkos::Experimental::local_deep_copy?
My comments about the Experimental namespace were not intended to create a reimplementation that we need to maintain, but, rather, to encourage Kokkos to promote local_deep_copy quickly.